_includes/footer.html

```diff
@@ -21,7 +21,8 @@
 </footer>
 
 <!-- jquery -->
-<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.1.1/jquery.min.js" integrity="sha384-3ceskX3iaEnIogmQchP8opvBy3Mi7Ce34nWjpBIwVTHfGYWQS9jwHDVRnpKKHJg7" crossorigin="anonymous"></script>
+<script>(window.jQuery) || document.write('<script src="/scripts/jquery.3.1.1.js"><\/script>');</script>
```

Comment:
The only thing I'm wondering is if this needs to be `src="{{ site.baseurl }}/scripts/jquery.3.1.1.js"`, but I don't really know enough about jekyll to answer that for sure

Reply:
I've added it.